app/models/container_group.rb

```diff
@@ -10,9 +10,12 @@ class ContainerGroup < ActiveRecord::Base
            :through => :container_definitions
   has_many :container_definitions, :dependent => :destroy
   belongs_to  :ext_management_system, :foreign_key => "ems_id"
-  has_many :labels, -> { where(:section => "labels") }, :class_name => CustomAttribute, :as => :resource, :dependent => :destroy
-  has_many :node_selector_parts, -> { where(:section => "node_selectors") }, :class_name => "CustomAttribute", :as => :resource, :dependent => :destroy
-  has_many :container_conditions, :class_name => ContainerCondition, :as => :container_entity, :dependent => :destroy
+  has_many :labels, -> { where(:section => "labels") }, :class_name => CustomAttribute,
+           :as => :resource, :dependent => :destroy
+  has_many :selector_parts, -> { where(:section => "selectors") }, :class_name => CustomAttribute,
+           :as => :resource, :dependent => :destroy
+  has_many :container_conditions, :class_name => ContainerCondition,
+           :as => :container_entity, :dependent => :destroy
```

Comment:
@yaacov is this just different indentation? Please don't mix these changes because it's hard to understand what you're changing here.